3_javascript/javascript2/week1/main.js

```diff
@@ -0,0 +1,22 @@
+console.log("Script loaded");
+
+const products = getAvailableProducts();
+console.log(products);
+
+
+const ul = document.querySelector("#productList")
+
+function renderProducts(products) {
+products.forEach(product => {
+    const li = document.createElement("li");
+    li.innerHTML = 
+     `<strong>${product.title}</strong><br>
+     Price: ${product.price}<br>
```

Comment:
if you look at the products in the helpers you can see it has a name, price, and rating (and no title). So, you have to change "title" to "name" otherwise you will get the title "undefined" for each.